ActindoWorkFlow/Operation/01_ManageWorkflows.md

```diff
@@ -0,0 +1,423 @@
+[!!Workflows](ActindoWorkFlow)
+
+# Manage the workflows
+
+The *Workflows* module allows the customers to define their own business processes, such as orders, shipments, returns, etc.
+The workflows can be created individually in the workflow editor, they can be edited subsequently and by publishing and unpublishing a workflow, the usage can be controlled. Further, obsolete workflows can be deleted.
+
+
+## Create a workflow
+
+Create a workflow to define a new process.
+
+### Prerequisites
+
+No prerequisites to fulfill.
+
+### Procedure
+
+*Workflows > Workflows > Tab OVERVIEW*
+
+![Workflows](/Assets/Screenshots/ActindoWorkFlow/Workflows/Workflows.png "[Workflows]")
+
+1. Click the ![Add](/Assets/Icons/Plus01.png "[Add]") (Add) button in the bottom right corner.   
+  The *New Workflow* window is displayed.
+
+  ![New workflow](/Assets/Screenshots/ActindoWorkFlow/Workflows/NewWorkflow.png "[New workflow]")
+
+2. Enter a name for the new workflow in the *Select a name for your new workflow* field.
+
+3. Enter a key for the workflow in the *Select a unique key for your new workflow* field. The key is required for API access and must be unique within the workflow version.
+
+4. Click the *Choose the data type of your start place* field and enter the name of the place or a keyword you are searching for.
+
+  > [Info] The list of places is filtered for your keyword as you type.
+
+5. Click the start place you want to add in the list of places.    
+
+6. Click the *Choose the data type of your end place* field and enter the name of the place or a keyword you are searching for.
+
+  > [Info] The list of places is filtered for your keyword as you type.
+
+7. Click the end place you want to add in the list of places.  
+
+8. Click the [CREATE] button in the bottom right corner.   
+  The new workflow is created. The *New Workflow* window is closed. The workflow editor with the defined start and end places is displayed.  
+
+  ![Workflow editor new](/Assets/Screenshots/ActindoWorkFlow/Workflows/WorkflowEditorNew.png "[Workflow editor new]")
+
+
+#### Create a transition
+
+1. Click the ![Add](/Assets/Icons/Plus04.png "[Add]") (Add) button next to the place or click the [NEW ACTION] button on the right hand above the workflow editor.     
+  A window to search for a transition is displayed.
+
+  ![Search action](/Assets/Screenshots/ActindoWorkFlow/Workflows/SearchAction.png "[Search action]")
+
+2. Click the *search for an action* field and enter the name of the transition or a keyword you are searching for. If you have clicked the ![Add](/Assets/Icons/Plus04.png "[Add]") (Add) button next to a place, only those transitions that are compatible with the data type of the place are displayed in the list.
```

Comment:
In Ui ist es klein, aber ich habe es generell großgeschrieben (Verweise). Einheitlich machen?